src/integration-test/java/com/commercetools/sync/integration/ctpprojectsource/categories/updateactionutils/ChangeParentIT.java

```diff
@@ -42,10 +42,12 @@ static void setup() {
         deleteAllCategories(CTP_SOURCE_CLIENT);
```

Comment:
Those ITs that you changed is removed in master, so you need to sync branch with master

Reply:
done !!